src/components/molecules/Footer/index.js

```diff
@@ -9,17 +9,19 @@ function Footer({
 }) {
   return (
     <div className="footer">
-      <Title title={textFooter} titleVariant="subtitle--small--black" />
-      <div className="">
-        {iconsFooter && iconsFooter.map((item, index) => (
-          <a href="#" key={item.id}>
-            <Icon
-              iconVariant="rounded"
-              src={iconsFooter && require(`../../../images/${iconsFooter[index].src}`).default}
-              alt={iconsFooter && iconsFooter[index].alt}
-            />
-          </a>
-        ))}
+      <div className="footer__icons">
+        <Title title={textFooter} titleVariant="subtitle--small--black" />
```

Comment:
The naming here `subtitle--small--black` looks a bit unconventional IMO.

Reply:
Why? There are 2 subtitle modificators: small and black.